scripts/benchmark_table.py

```
@@ -0,0 +1,409 @@
+Run from the project root with:
+    python scripts/benchmark_table.py <output> [baseline]
+
+Arguments:
+    output      The directory the harness wrote its reports into
+    baseline    The recorded run to compare against, optional
+"""
+
+from json import dumps, loads
+from os import listdir
+from os.path import exists, isdir, join
+from sys import argv, exit
```

Ruff (0.16.2)

[error] Import `exit` is shadowing a Python builtin

(A004)